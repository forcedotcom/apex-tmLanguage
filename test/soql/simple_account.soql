-- SYNTAX TEST "source.soql" "simple testcase"

   SELECT Id, Name FROM Account
-- ^^^^^^ source.soql keyword.operator.query.select.apex
--        ^^ source.soql keyword.query.field.apex
--          ^ source.soql punctuation.separator.comma.apex
--            ^^^^ source.soql keyword.query.field.apex
--                 ^^^^ source.soql keyword.operator.query.from.apex
--                      ^^^^^^^ source.soql storage.type.apex
