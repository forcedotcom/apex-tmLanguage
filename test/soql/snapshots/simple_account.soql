
SELECT Id, Name FROM Account

