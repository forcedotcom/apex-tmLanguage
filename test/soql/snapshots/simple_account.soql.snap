>
>SELECT Id, Name FROM Account
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^ source.soql keyword.query.field.apex
#         ^ source.soql punctuation.separator.comma.apex
#          ^ source.soql
#           ^^^^ source.soql keyword.query.field.apex
#               ^ source.soql
#                ^^^^ source.soql keyword.operator.query.from.apex
#                    ^ source.soql
#                     ^^^^^^^ source.soql storage.type.apex
>
>